File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {
        int max_v = 0;
        int max_h = 0;
            
        for (int[] i : sizes){
            max_v = Math.max(Math.max(i[0], i[1]), max_v);
            max_h = Math.max(Math.min(i[0], i[1]), max_h);
        }

        return max_v * max_h;
    }
}

File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.kt
class Solution {
    fun solution(sizes: Array<IntArray>): Int {
        for(i in 0 until sizes.size){
            if(sizes[i][0] < sizes[i][1]){
                var a = sizes[i][0]
                sizes[i][0] = sizes[i][1]
                sizes[i][1] = a
            }
        }
        
        var b = sizes[0][0]
        var c = sizes[0][1]
        
        for ( i in 1 until sizes.size) {
            if(b<sizes[i][0]){
                b = sizes[i][0]
            }
            if(c<sizes[i][1]){
                c = sizes[i][1]
            }
        }
        
        var answer: Int = 0
        return b*c
    }
}